rhdh-operator/README.md

```diff
@@ -0,0 +1,36 @@
+# Red Hat Developer Hub Operator
```

Comment:
Please rename the folder to either "developer-hub-operator"  or "redhat-developer-hub-operator"

I would like to avoid the rhdh acronym since it is so new and if someone is looking for it in the catalog they may not know the acronym yet.

Reply:
Done mate